cypress.config.ts

```diff
@@ -0,0 +1,45 @@
+// eslint-disable-next-line import/no-extraneous-dependencies
+import { defineConfig } from "cypress"
+import fs from "fs"
+
+export default defineConfig({
+  e2e: {
+    experimentalMemoryManagement: true,
+    video: true,
+    baseUrl: "http://localhost:3000",
+    userAgent: "cypress",
+    specPattern: "./cypress/e2e/**/*.spec.ts",
+    defaultCommandTimeout: 15_000,
+    env: {
+      guildApiUrl: "https://api.guild.xyz/v2",
+      userAddress: "0x304Def656Babc745c53782639D3CaB00aCe8C843",
+      platformlessGuildName: "Platformless Cypress Gang",
+      platformlessGuildUrlName: "platformless-cypress-gang",
+      guildName: "Cypress Gang",
+      guildUrlName: "cypress-gang",
+      dcClientId: "868172385000509460",
+      dcServerId: "1096417797292171365", // We'll delete the created roles in this Discord server
+      tgId: "-1001653099938",
```

Comment:
we don't need these, but left them here in case we'll need them for testing guild creation with platforms